lib/fs.js

```diff
@@ -101,6 +101,10 @@ function nullCheck(path, callback) {
   return true;
 }
 
+function isFd(path) {
+  return (path >>> 0) === path && path >= 0;
```

Comment:
You seem to be correct. This is probably residue from [this discussion](/nodejs/node-v0.x-archive/pull/8522#discussion-diff-19893851).